modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/communicators/cbc_async_comm.h

```diff
@@ -51,6 +53,9 @@ class CBC_ASynchronousCommunicator : public AsynchronousCommunicator
 protected:
   const size_t angle_set_id_;
   CBC_FLUDS& cbc_fluds_;
+
+  // location_id, cell_global_id, face_id
+  typedef std::tuple<int, uint64_t, unsigned int> MessageKey;
```

Comment:
This is not in code standard yet, but the preference is to use `using` over `typedef` (see #86), so:
```
using MessageKey = std::tuple<int, uint64_t, unsigned int>;
```

Reply:
Fixed.